_posts/jared_baker/2017-10-20-Day-1-Collaboratory.md

```diff
@@ -0,0 +1,276 @@
+
+From the Access & Security, Security Groups tab, click Manage rules
+
+<figure>
+    <img id="secgroup-manage" src="{{site.urlimg}}jared_baker/day1collab/5-secgroup-manage.png" data-featherlight="#secgroup-manage" />
+</figure>
+
+Add a rule for SSH from the drop down. Provide a range of IP addresses to allow SSH FROM, such as your corporate IP space or alternatively you can leave it open to all (0.0.0.0/0 - less secure)
+
+<figure>
+    <img id="secgroup-add" src="{{site.urlimg}}jared_baker/day1collab/5-secgroup-add.png" data-featherlight="#secgroup-add" />
+</figure>
+
+<a id="6"></a>
+## Step 6: SSH to the instance
+It's time to SSH to your instance.
+
+You will need:
+- The floating IP address of the instance
+- Your SSH private key (the .pem downloaded in step 2)
+- The username, which is standardized based on what OS/Image used (ubuntu, centos, debian)
+
+~~~bash
+ssh -i <key.pem> <username>@<floating-ip-address>
+ssh -i collaboratory-key.pem ubuntu@142.1.177.144
+~~~
+
+For Windows + Putty users, you will need to convert the .pem to .ppk. See instructions [here](https://github.com/davidheijkamp/docs/wiki/Howto:-Creating-and-using-OpenStack---SSH-keypairs-on-Windows)
+
+<a id="7"></a>
+## Step 7: Install icgc-storage-client
+The ICGC storage client is an application that allows you to download ICGC data from Collaboratory's protected object storage. Its available as a stand alone client or docker container. [More detailed instructions here](http://docs.icgc.org/cloud/guide/#installation)
+
+First, install the prerequisite java packages on your instance
+
+~~~bash
+sudo add-apt-repository ppa:webupd8team/java
+sudo apt-get update
+sudo apt-get install oracle-java8-installer
+~~~
+
+Download & Extract the icgc-storage-client
+
+~~~bash
+wget -O icgc-storage-client.tar.gz https://dcc.icgc.org/api/v1/ui/software/icgc-storage-client/latest
+tar -xvzf icgc-storage-client.tar.gz
+~~~
+
+Ensure the client runs
+*(The binary is located in the 'bin' directory to where it was extracted)*
+
+~~~bash
+./icgc-storage-client --version
+
+> ICGC Storage Client
+
+  Version: 1.0.23
+  Built:   10.01.2017 @ 15:03:36 EST
+  Contact: dcc-support@icgc.org
+~~~
+
+<a id="8"></a>
+## Step 8: Configure icgc-storage-client
+
+First obtain or generate your token by logging in to the [The ICGC Data Portal](https://dcc.icgc.org)
+
+<figure>
+    <img id="icgc-logged" src="{{site.urlimg}}jared_baker/day1collab/8-icgc-logged-in.png" data-featherlight="#icgc-logged" />
+</figure>
+
+<figure>
+    <img id="icgc-token" src="{{site.urlimg}}jared_baker/day1collab/8-icgc-tokenmgr.png" data-featherlight="#icgc-token" />
+</figure>
+
+<figure>
+    <img id="icgc-tokenlist" rc="{{site.urlimg}}jared_baker/day1collab/8-icgc-tokenlist.png" data-featherlight="#icgc-tokenlist" />
+</figure>
+
+Copy your token ID and paste it in to the accessToken line in 'application.properties' in the 'conf' directory of the icgc-storage client. Save and exit!
```

Comment:
Add a note about changing the "transport.memory" and "transport.parallel" to use x GB of RAM per CPU available, so if the instance has 8 cores and 48 GB of RAM, then use these settings while allowing one core and sufficient RAM for the operating system:

transport.memory=6
transport.parallel=7

This will allow 7 parallel download streams with each using 6 GB of RAM to improve the download speed.